modelplugin/aether-2.0.0/aether.tree

```diff
@@ -0,0 +1,61 @@
+module: access-profile
+  +--rw access-profile
+     +--rw access-profile* [id]
+        +--rw id              access-profile-id
+        +--rw display-name?   string
+        +--rw type?           string
+        +--rw filter?         string
+        +--rw description?    string
+module: aether-subscriber
+  +--rw subscriber
+     +--rw ue* [id]
+        +--rw id                       ietf:uuid
+        +--rw display-name?            string
+        +--rw (imsi)?
```

Comment:
I think we did say in the Monday call that there would be a tree to handle each access technology. So there might be a "5G" or "ran" container here before you get in to IMSI & plmn etc

Reply:
I was thinking about deferring this until we have more information -- my concern is that there may be overlap between the multiple uses and I don't want to repeat the same piece of information in two or more subtrees. Let's take this in stages, we'll look at what information there is in SD-Ran next. Then we also need to look at what keys will be added for 5G that aren't part of 4G.

Reply:
sounds good - we can tackle it later then